src/pages/MainPage/PrivateRoomModal.tsx

```diff
@@ -0,0 +1,60 @@
+import * as Dialog from '@radix-ui/react-dialog';
+import { Cross2Icon, PaperPlaneIcon } from '@radix-ui/react-icons';
+import { ComponentProps, ReactNode, useState } from 'react';
+import Input from '@/common/Input/Input';
+
+interface PrivateRoomModalProps {
+  children: ReactNode;
+  className?: ComponentProps<'div'>['className'];
+}
+
+const wait = () => new Promise((resolve) => setTimeout(resolve, 2000));
+
+const PrivateRoomModal = ({ children, className }: PrivateRoomModalProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  return (
+    <Dialog.Root
+      open={isOpen}
+      onOpenChange={setIsOpen}>
+      <Dialog.Trigger>{children}</Dialog.Trigger>
+      <Dialog.Portal>
+        <Dialog.Overlay
+          className={
+            'fixed top-1/2 left-1/2 translate-x-[-50%] translate-y-[-50%] w-[100vw] h-[100vh] bg-black/30 animate-[overlayShow_150ms_cubic-bezier(0.16,1,0.3,1)]'
+          }
+        />
+        <Dialog.Content
+          className={` rounded-[1rem] border-green-100 border-[0.3rem] w-[30rem] h-[15rem] flex flex-col items-center justify-center bg-white z-10 fixed top-1/2 left-1/2 translate-x-[-50%] translate-y-[-50%] ${className}`}>
+          <Dialog.Title>비밀번호를 입력해주세요!</Dialog.Title>
+          <Dialog.Description>
+            <form
+              className='flex gap-[1rem] items-center'
+              onSubmit={(e) => {
+                e.preventDefault();
+                wait().then(() => setIsOpen(false));
+              }}>
+              <Input whSize='w-[20rem] h-[3rem]' />
+              <button
+                type='submit'
+                className='hover:bg-gray-100 rounded-[0.3rem]'>
+                <PaperPlaneIcon
+                  color='green'
+                  className='size-[2rem]'
+                />
+              </button>
+            </form>
+          </Dialog.Description>
```

Comment:
Dialog.Description이 p태그라 이런 경고가 뜨네용 ㅠㅠ
<img width="700" alt="스크린샷 2024-02-19 오전 11 15 15" src="https://github.com/Team-E2I4/Team-E2I4-TiKiTaza-FE/assets/66900221/eb78ecb9-f673-4666-97ee-530dfc3a8ef5">

```suggestion
            <form
              className='flex gap-[1rem] items-center'
              onSubmit={(e) => {
                e.preventDefault();
                wait().then(() => setIsOpen(false));
              }}>
              <Input whSize='w-[20rem] h-[3rem]' />
              <button
                type='submit'
                className='hover:bg-gray-100 rounded-[0.3rem]'>
                <PaperPlaneIcon
                  color='green'
                  className='size-[2rem]'
                />
              </button>
            </form>
```


<form>태그가 Dialog.Description의 자식이 아니라 Dialog.Content의 자식으로 들어가면 될 것 같습니다!!

Reply:
오우 감사합니다 수정반영하였습니다ㅎㅎ